recipes/libtool/all/conanfile.py

```diff
@@ -57,17 +59,25 @@ def _configure_autotools(self):
         conf_args = [
             "--datarootdir={}".format(datarootdir),
             "--prefix={}".format(prefix),
-            "--disable-shared",
-            "--disable-static",
+            "--enable-shared",
+            "--enable-static",
             "--disable-ltdl-install",
         ]
         self._autotools.configure(args=conf_args, configure_dir=self._source_subfolder)
         return self._autotools
 
+    def _patch_sources(self):
+        # FIXME: tools.patch cannot patch read-only files on Windows
+        os.chmod(os.path.join(self._source_subfolder, "build-aux", "ltmain.sh"), 0o777)
```

Comment:
oh, interesting... did you create conan issue for that?

Reply:
See https://github.com/conan-io/conan/issues/6676